src/components/Questionnaire/QuestionnaireForm.tsx

```diff
@@ -158,11 +158,13 @@ export function QuestionnaireForm({
         (error: QuestionValidationError | DetailedValidationError) => {
           // Handle question-specific errors
           if ("question_id" in error) {
-            form.errors.push({
+            form?.errors.push({
```

Comment:
let's not enter the `forEach` loop in line 154 in the first place if the form is nullish